pvlib/solarposition.py

```diff
@@ -194,7 +194,7 @@ def spa_c(time, latitude, longitude, pressure=101325, altitude=0,
                                 hour=date.hour,
                                 minute=date.minute,
                                 second=date.second,
-                                timezone=0,  # date uses utc time
+                                time_zone=0,  # date uses utc time
```

Comment:
Add a comment above, just below the link to the spa code, that explains that `timezone` is replaced with `time_zone` in `spa.c` during the `pvlib` build process.

Reply:
added [this note](../../../mikofski/pvlib-python/blob/fix_spa_c_files_timezone_compile_fail/pvlib/solarposition.py#L165-L167)
>Note: The ``timezone`` field in the SPA C files is replaced with ``time_zone`` to avoid a nameclash with the function ``__timezone`` that is redefined by Python-3.5. This issue is `Python bug 24643 <https://bugs.python.org/issue24643>`_.